Kiosk/Auction Listings/MasonryCollectionViewCell.swift

```diff
@@ -1,4 +1,6 @@
 import UIKit
+import RxSwift
+import ORStackView
```

Comment:
The basic idea in this class is to remove a bunch of autolayout code and replace it with `ORTaggedBasedAutoStackView` instead, removing and re-adding labels as necessary.